### ISSUE TITLE
Broken File -> Import file button

### ISSUE THREAD

**Issue by AlbeRCLife** [swagger-api/swagger-editor#1718]
| Q                                   | A
| ----------------------------------- | -------
| Bug or feature request?             |  Bug
| Which Swagger/OpenAPI version?      | N/A
| Which Swagger-Editor version?       | latest (v3.5.1)
| How did you install Swagger-Editor? | Docker image "swaggerapi/swagger-editor"
| Which browser & version?            | Google Chrome Versión 65.0.3325.181 (Build oficial) (64 bits)
| Which operating system?             | Windows 10 Enterprise

### Demonstration API definition

```yaml
your: "API definition goes here"
```
It really doesn't matter. Just initiated the docker image.

### Configuration (browser query string, constructor, config.yaml)

```js
{
  "your": { "constructorConfig": "here" }
}
```

`?yourQueryStringConfig=here`

It really doesn't matter. Just initiated the docker image.

### Expected Behavior

Expect a modal to pop up with the option to load a file in the editor.

### Current Behavior

Nothing happens, console outputs error:
![image](https://user-images.githubusercontent.com/16323399/37907532-aa939b06-3105-11e8-8670-dc2b0735d1c4.png)

### Possible Solution

I changed this things to make it work:

In the js loaded from the browser:
/dist/swagger-editor-standalone-preset.js
line 1:20598 or search for '[e||"modal"]', there will be 2 matches. Change that to this: 
```javascript
[e !== undefined && typeof e === "string" ? e:"modal"]
```

Before .show and .hide (both matches).

If you go to the source code here it would be this way:

Go to /src/standalone/topbar/topbar.jsx and edit lines 252 and 257 from
[name || "modal"] 
to
[name !== undefined && typeof name === "string" ? name : "modal" ]

Of course it can be done better I guess, but that worked for me.
Please note that it seems that below version 3.5.0 this issue wasn't there.
### Context

I just tried to import a yaml the old fashion way via menu instead of drag and drop and it just exploded.

**Comment by hkosova** [comment]
See #1714. This was fixed in v.3.5.1, but it looks like the online editor was not updated yet.

**Comment by rupe120** [comment]
When will this be applied to the online editor?

**Comment by shockey** [comment]
Sorry for the delay - there were some issues with the Docker image builder, which was holding up the Swagger.io update.

Everything's resolved now, 3.5.1 is live at editor.swagger.io :smile:
